Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -1067,13 +1078,15 @@ SuperfluousWordList: !list
     - maybe
     - say
     - like
-DurationDateRestrictions: [ vandaag, nu ]
 # For DurationDateRestrictions, only translate these terms if this particular circumstance applies in the target language. If not, keep it empty. 
+DurationDateRestrictions: [ today, now ]
```

Comment:
_For DurationDateRestrictions, only translate these terms if this particular circumstance applies in the target language. If not, keep it empty._ 

If this doesn't apply, keep them empty. Otherwise revert the changes in translation.

Reply:
@iMicknl & @STayinloves: question, if there is a regex from English that is not used in Dutch or other languages should it be removed from the respective yaml?

Reply:
@iMicknl so this should be changed to DurationDateRestrictions: []

Reply:
Yes 👍